app/src/main/java/nl/tudelft/b_b_w/model/Block.java

```diff
@@ -15,24 +15,28 @@
     private String publicKey;
     private String iban;
     private boolean isRevoked;
+    private int trustValue;
 
     /**
+     * Constructor for a block
      * @param _owner              owner of a block
      * @param _ownHash            our own hash
      * @param _previousHashChain  the hash value of the block before in the chain
      * @param _previousHashSender the hash value of the block before of the sender
      * @param _publicKey          public key of the sender
      * @param _iban               IBAN number of a contact
+     * @param _trustvalue         the trust value of the contact
      * @param _isRevoked          boolean to check whether a block is revoked or not
      */
-    public Block(String _owner, String _ownHash, String _previousHashChain, String _previousHashSender, String _publicKey, String _iban, boolean _isRevoked) {
+    public Block(String _owner, String _ownHash, String _previousHashChain, String _previousHashSender, String _publicKey, String _iban, int _trustvalue, boolean _isRevoked) {
```

Comment:
This method signature becomes unbelievably large, maybe refactor it in the future.